diff --git a/node_modules/react-router/dist/development/chunk-5KNZJZUH.mjs b/node_modules/react-router/dist/development/chunk-5KNZJZUH.mjs
index 96185d2..355dbbd 100644
--- a/node_modules/react-router/dist/development/chunk-5KNZJZUH.mjs
+++ b/node_modules/react-router/dist/development/chunk-5KNZJZUH.mjs
@@ -8550,13 +8550,7 @@ async function loadRouteModule(route, routeModulesCache) {
     return routeModulesCache[route.id];
   }
   try {
-    let routeModule = await import(
-      /* @vite-ignore */
-      /* webpackIgnore: true */
-      route.module
-    );
-    routeModulesCache[route.id] = routeModule;
-    return routeModule;
+    throw new Error("Route modules are not supported by older devices.");
   } catch (error) {
     console.error(
       `Error loading route module \`${route.module}\`, reloading page...`
@@ -8923,11 +8917,7 @@ function createClientRoutes(manifest, routeModulesCache, initialState, ssr, isSp
       return fetchServerHandler(singleFetch);
     }
     function prefetchModule(modulePath) {
-      import(
-        /* @vite-ignore */
-        /* webpackIgnore: true */
-        modulePath
-      );
+      throw new Error("Route modules are not supported by older devices.");
     }
     function prefetchRouteModuleChunks(route2) {
       if (route2.clientActionModule) {
@@ -9070,11 +9060,7 @@ function createClientRoutes(manifest, routeModulesCache, initialState, ssr, isSp
       }
       dataRoute.lazy = {
         loader: route.hasClientLoader ? async () => {
-          let { clientLoader } = route.clientLoaderModule ? await import(
-            /* @vite-ignore */
-            /* webpackIgnore: true */
-            route.clientLoaderModule
-          ) : await getLazyRoute();
+          throw new Error("Route modules are not supported by older devices.");
           invariant2(clientLoader, "No `clientLoader` export found");
           return (args, singleFetch) => clientLoader({
             ...args,
@@ -9085,11 +9071,7 @@ function createClientRoutes(manifest, routeModulesCache, initialState, ssr, isSp
           });
         } : void 0,
         action: route.hasClientAction ? async () => {
-          let clientActionPromise = route.clientActionModule ? import(
-            /* @vite-ignore */
-            /* webpackIgnore: true */
-            route.clientActionModule
-          ) : getLazyRoute();
+          throw new Error("Route modules are not supported by older devices.");
           prefetchRouteModuleChunks(route);
           let { clientAction } = await clientActionPromise;
           invariant2(clientAction, "No `clientAction` export found");
@@ -9102,11 +9084,7 @@ function createClientRoutes(manifest, routeModulesCache, initialState, ssr, isSp
           });
         } : void 0,
         middleware: route.hasClientMiddleware ? async () => {
-          let { clientMiddleware } = route.clientMiddlewareModule ? await import(
-            /* @vite-ignore */
-            /* webpackIgnore: true */
-            route.clientMiddlewareModule
-          ) : await getLazyRoute();
+          throw new Error("Route modules are not supported by older devices.");
           invariant2(clientMiddleware, "No `clientMiddleware` export found");
           return clientMiddleware;
         } : void 0,
